src/tasks/DockerBuildTaskProvider.ts

```diff
@@ -96,7 +100,7 @@ export class DockerBuildTaskProvider implements TaskProvider {
             .create('docker', 'build', '--rm')
             .withFlagArg('--pull', options.pull)
             .withNamedArg('-f', options.dockerfile)
-            .withKeyValueArgs('--build-arg', options.args)
+            .withKeyValueArgs('--build-arg', options.buildArgs)

```

Comment:
This change means users of today's debugging configuration cannot just copy/paste their existing configuration into a task; they'll need to modify the property name?  (Not a bad thing, necessarily, but could complicate the migration story.)

Reply:
Yeah, that is one effect. I was confused by `args` myself, I thought it was a way to pass arbitrary extra arguments to `docker build`, not a way of controlling the `--build-args`.

Now is the only opportunity to make a change like this so I'm inclined to do it even though it will impact the subset of users who want to copypasta the config from launch to tasks.